src/plugins/unified_search/public/search_bar/search_bar.tsx

```diff
@@ -615,14 +618,6 @@ class SearchBarUI<QT extends (Query | AggregateQuery) | Query = Query> extends C
     );
   }
 
-  private hasFiltersOrQuery() {
-    const hasFilters = Boolean(this.props.filters && this.props.filters.length > 0);
-    const hasQuery = Boolean(
-      this.state.query && isOfQueryType(this.state.query) && this.state.query.query
-    );
-    return hasFilters || hasQuery;
-  }
```

Comment:
ℹ️ Dropped this because it was only being passed to `SavedQueryManagementList` which wasn't using it anyway.